bundles/CoreBundle/src/DependencyInjection/Configuration.php

```diff
@@ -860,20 +860,6 @@ private function addDocumentsNode(ArrayNodeDefinition $rootNode): void
                             ->info('Optionally define route patterns to lookup static pages. Regular Expressions like: /^\/en\/Magazine/')
                         ->end()
```

Comment:
Is here a `end()` missing?
```suggestion
                        ->end()
                    ->end()
```

Reply:
Thank you! 
It seems that i need to remember to spot them 😄 

Weird how it works fine even without these and at the same time there are no tools like in phpstorm, phpstan or cs-fixer/beautify/autocomplete to fix indentation and missing `end()`, there are still many inconsistencies and mistakes.

In this case i was just reverting the [changes ](https://github.com/pimcore/pimcore/pull/14175/files?w=1#diff-bab0da2e129502ead20a1f1ac979fd2260c5d4c8da202b6a8fbf1e9d906d4541R905-R918) so it was missing this `end()` even before that.